backend/handlers/handlers.go

```diff
@@ -204,10 +205,13 @@ func NewHandlers(
 	getAPIRouter(apiRouter)("/socksproxy/check", handlers.postSocksProxyCheck).Methods("POST")
 	getAPIRouter(apiRouter)("/exchange/moonpay/buy-supported/{code}", handlers.getExchangeMoonpayBuySupported).Methods("GET")
 	getAPIRouter(apiRouter)("/exchange/moonpay/buy/{code}", handlers.getExchangeMoonpayBuy).Methods("GET")
+	getAPIRouter(apiRouter)("/exchange/pocket/buy/{code}", handlers.getExchangePocketBuy).Methods("GET")
+	getAPIRouter(apiRouter)("/exchange/pocket/supported/{code}", handlers.getExchangePocketSupported).Methods("GET")
 	getAPIRouter(apiRouter)("/aopp", handlers.getAOPPHandler).Methods("GET")
 	getAPIRouter(apiRouter)("/aopp/cancel", handlers.postAOPPCancelHandler).Methods("POST")
 	getAPIRouter(apiRouter)("/aopp/approve", handlers.postAOPPApproveHandler).Methods("POST")
 	getAPIRouter(apiRouter)("/aopp/choose-account", handlers.postAOPPChooseAccountHandler).Methods("POST")
+	getAPIRouter(apiRouter)("/pocket-addr-sign", handlers.postPocketWidgetSignAddress).Methods("POST")
```

Comment:
nit: we usually spell out words: address not addr.

super-nit: `address-sign` vs `SignAddress` - why the different order of the words? :smile: 

could also do `/pocket/...` in case more endpoints should come.

Reply:
Ok! about the super-nit: I think I changed it in a second moment and didn't notice the discrepancy :D